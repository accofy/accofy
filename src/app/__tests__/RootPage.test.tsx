import '@testing-library/jest-dom';

import { render } from '@testing-library/react';

import RootPage from '@/app/page';

describe('Root layout', () => {
  it('snapshot testing', () => {
    const { container } = render(<RootPage />);
    expect(container).toMatchSnapshot();
  });
});
