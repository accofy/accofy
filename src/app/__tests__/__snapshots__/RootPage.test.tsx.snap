// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root layout snapshot testing 1`] = `
<div>
  <main
    class="flex min-h-screen flex-col items-center justify-center gap-6 bg-black text-white"
  >
    <h1
      class="text-4xl"
    >
      A simple Next js starter template
    </h1>
    <div
      class="flex gap-5"
    >
      <a
        class="rounded-md border border-white/10 px-20 py-16 text-3xl font-medium  transition-all duration-300 ease-in-out hover:border-white/5 hover:bg-white/10"
        href="/counter"
      >
        Counter
      </a>
      <a
        class="rounded-md border border-white/10 px-20 py-16 text-3xl font-medium  transition-all duration-300 ease-in-out hover:border-white/5 hover:bg-white/10"
        href="/about"
      >
        About
      </a>
    </div>
  </main>
</div>
`;
